src/worker.cc

```diff
@@ -293,7 +293,8 @@ rapidjson::Document from_string(const std::string& json, const valhalla_exceptio
 
 void add_date_to_locations(Options& options,
                            google::protobuf::RepeatedPtrField<valhalla::Location>& locations) {
-  if (options.has_date_time() && locations.size()) {
+  // otherwise we do what the person was asking for
+  if (options.has_date_time() && !locations.empty()) {
```

Comment:
new protobuf has `empty`. it also has a lot of the stuff we have in our protoconversions.. we should probably delete some of those